google-cloud/pom.xml

```diff
@@ -92,7 +92,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>google-cloud-vision</artifactId>
-      <version>${project.version}</version>
```

Comment:
I was making it consistent with the other beta packages in the repo. For example for logging same file has:
    <dependency>
      <groupId>${project.groupId}</groupId>
      <artifactId>google-cloud-logging</artifactId>
      <version>${beta.version}</version>
    </dependency>

Do we really want to have it alpha here for vision?